plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1480,3 +1480,49 @@ func (c *VolumeNodeChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMetad
 	}
 	return true, nil, nil
 }
+
+type NoDeletedPVCChecker struct {
+	pvcInfo PersistentVolumeClaimInfo
+}
+
+// NewPVCPredicate evaluates if a pod can fit due to the PVC it uses.
+func NewNoDeletedPVCsPredicate(pvcInfo PersistentVolumeClaimInfo) algorithm.FitPredicate {
```

Comment:
Could you please check if this predicate need to be handled seperatedly according to: https://github.com/jsafrane/kubernetes/blob/3835234dd8c78d9955d7e3e06cafaa7197d6a31f/plugin/pkg/scheduler/algorithm/predicates/predicates.go#L71-L77

Reply:
fixed `invalidatePredicatesForPvc`